laghos_rom.cpp

```diff
@@ -168,8 +194,19 @@ void BasisGeneratorFinalSummary(CAROM::SVDBasisGenerator* bg, const double energ
     cout << "Take first " << cutoff << " of " << sing_vals->numColumns() << " basis vectors" << endl;
 }
 
+void printSnapshotTime(std::vector<double> const &tSnap, std::string const filename)
+{
+    cout << tSnap.size() << endl;
```

Comment:
Splitting the `cout` between two functions is a little trickier than it needs to be. You could just pass the variable name as a string argument. This would also eliminate several `cout` calls in `ROM_Sampler::Finalize`.